src/ticketmap/index.tsx

```diff
@@ -496,6 +499,16 @@ export default class TicketMap extends Component<Props & DefaultProps, State> {
     }
   }
 
+  clearSelection = () => {
```

Comment:
There's already a function for this, `deselectSection` clears all sections when no sectionId is passed into it.

Reply:
Is that a feature?

Reply:
That was designed for the public API